rockr/views.py

```diff
@@ -93,7 +94,63 @@ def user_goals(user_id):
         ).first()
         db_manager.delete(ug)
         return format_response(204, None)
-
+    
+#Added for the User Band queries
```

Comment:
this comment doesn't add any value. I should be able to tell what the endpoint is for based on the definition